src/packaging/debian/cassandra-reaper.service

```diff
@@ -21,6 +21,7 @@ After=remote-fs.target syslog.socket
 [Service]
 User=reaper
 Group=reaper
+WorkingDirectory=/usr/share/cassandra-reaper
 ExecStart=/usr/local/bin/cassandra-reaper
```

Comment:
patch looks good to me.

but i'm a little in the dark as to when reaper is in installed in `/usr/share` and when in `/usr/local` ?
even above we're referencing both in the same service file.

Reply:
can you ask your colleagues about installation path?
I cant find another install location in current debian package.
maybe on redhat-like os path may change?

Reply:
@adejanovski can you enlighten us about the different possible installation paths?

Reply:
@adejanovski Version 2.0.2 problem is still there.
Can you give answer for this question?

Reply:
We used to consider `/usr/share` only back in the 1.0 days, and @michaelsembwever introduced using `find` and checking both `/usr/share` and `/usr/local`.
The location will actually depend on the installation method.
If we can't look for both, we should only consider `/usr/share`.

Reply:
@michaelsembwever if I understood @adejanovski correctly, you can merge this pr, to master.